jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/InvalidJavaVersionException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.plugins.common;
+
+/** Thrown when attempting to use an incompatible base image. */
+public class InvalidJavaVersionException extends Exception {
```

Comment:
It looks like from the javadoc this should be called `IncompatibleBaseImageException`, or maybe `JavaVersionMismatchException`?